packages/gatsby-remark-images/README.md

```diff
@@ -44,17 +44,18 @@ plugins: [
+| `quality`              | `50`     | The quality level of the generated files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `withWebp`             | `false`  | Additionally generate WebP versions alongside your chosen file format. They are added as a srcset with the appropriate mimetype and will be loaded in browsers that support the format. Pass `true` for default support, or an object of options to specifically override those for the WebP files. For example, pass `{ quality: 80 }` to have the WebP images be at quality level 80.                                                                                                                                                                                                                                            |
+| `tracedSVG`            | `false`  | Use traced SVGs for placeholder images instead of the "blur up" effect. Pass `true` for traced SVGs with the default settings (seen [here][3]), or an object of options to override the defaults. For example, pass `{ color: "#F00", turnPolicy: "TURNPOLICY_MAJORITY" }` to change the color of the trace to red and the turn policy to TURNPOLICY_MAJORITY. See [`node-potrace` parameter documentation][4] for a full listing and explanation of the available options.                                                                                                                                                        |
+| `loading`              | `string` | Set the browser's native lazy loading attribute. One of `lazy`, `eager` or `auto`. Defaults to `lazy`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Not sure what is going on with the diff, I'll try to update the PR so that only this line shows as changed.